### PR TITLE
R doc 1843 studio backup

### PR DESCRIPTION
**Pull request by morhilai**

### CODE REVIEW

File: Documentation/5.1/Raven.Documentation.Pages/studio/database/tasks/backup-task.markdown [hunk 1]
```diff
@@ -0,0 +1,256 @@
...
+
+Select the content to back up. Note: Both types can be scheduled.  
+
+  1. **Full Backup** 
+     Full Backup will back up _all_ the database data every time the task is scheduled to work  
+
+  2. **Incremental Backup** 
+     Incremental Backup will only back up the delta of the data since the last Backup that has occurred  
+
+Schedule the Backup Task time using a [Cron Expression](http://www.quartz-scheduler.org/documentation/quartz-2.3.0/tutorials/crontrigger.html)
```
Comment (garayx) [inline Documentation/5.1/Raven.Documentation.Pages/studio/database/tasks/backup-task.markdown:112]: there is a time selector, cron expresion is available using custom option

File: Documentation/5.1/Raven.Documentation.Pages/studio/database/tasks/backup-task.markdown [hunk 2]
```diff
@@ -0,0 +1,256 @@
...
+![](images/backup-task-details.png "'Backups' View")
+
+{INFO: }
+1. **Backup Task Details**:  
+  *  Task Status - Active / Not Active / Not on Node  
+  *  Destinations - List of all backup destinations defined  
+  *  Last Full Backup - The last time a Full Backup was done 
+     (Snapshot / Backup type - depending on task definition)  
+  *  Last Incremental Backup - The last time an Incremental Backup was done  
+  *  Next Estimated Backup - Time for next backup
```
Comment (garayx) [inline Documentation/5.1/Raven.Documentation.Pages/studio/database/tasks/backup-task.markdown:190]: add 'Last executed on node' and explain the difference between the node tag near task name

File: Documentation/5.1/Raven.Documentation.Pages/studio/database/tasks/backup-task.markdown [hunk 3]
```diff
@@ -0,0 +1,256 @@
...
+
+{PANEL: When the Cluster or Node are Down}
+
+* **When the cluster is down** (and there is no leader):  
+
+  * Creating a _new_ Ongoing Task is a Cluster-Wide operation,  
+    thus, a new Ongoing Backup Task ***cannot*** be scheduled.  
+
+  * If a Backup Task was _already_ defined and active when the cluster went down,  
+    then the Backup Task will still ***continue to execute*** on its defined schedule (on its responsible node).
```
Comment (garayx) [inline Documentation/5.1/Raven.Documentation.Pages/studio/database/tasks/backup-task.markdown:211]: add link to `server/clustering/distribution/highly-available-tasks#responsible-node`

Comment (morhilai) [reply Documentation/5.1/Raven.Documentation.Pages/studio/database/tasks/backup-task.markdown:211]: Done

File: Documentation/5.1/Raven.Documentation.Pages/studio/database/tasks/backup-task.markdown (hunk 2 above)
Comment (morhilai) [reply Documentation/5.1/Raven.Documentation.Pages/studio/database/tasks/backup-task.markdown:190]: Done

File: Documentation/5.1/Raven.Documentation.Pages/studio/database/tasks/backup-task.markdown (hunk 1 above)
Comment (morhilai) [reply Documentation/5.1/Raven.Documentation.Pages/studio/database/tasks/backup-task.markdown:112]: Fixed

**Review by garayx — APPROVED** [review]

**Comment by garayx** [comment]
we can open a separate issue to update the images for updated views

**Pull request merged**